components/RegistryInstructions.tsx

```diff
@@ -207,12 +214,24 @@ function RegistryInstructions(props: { isOpen: boolean; close: () => void }) {
                           </label>
                           <input
                             id="subdirectory"
-                            className={`block w-full px-4 py-2 my-1 leading-normal bg-white border border-gray-200 rounded-lg outline-none shadow hover:shadow-sm focus:shadow-sm appearance-none focus:border-gray-300 hover:border-gray-300 mt-1`}
+                            className={`block w-full px-4 py-2 my-1 leading-normal bg-white border border-gray-200 rounded-lg outline-none shadow hover:shadow-sm focus:shadow-sm appearance-none focus:border-gray-300 hover:border-gray-300 mt-1 ${
+                              isSubdirectoryValid === true
+                                ? "border-green-300 hover:border-green-300 focus:border-green-300"
+                                : isSubdirectoryValid === false
+                                ? "border-red-300 hover:border-red-300 focus:border-red-300"
+                                : ""
+                            }`}
                             type="text"
                             placeholder="Subdirectory"
                             value={subdirectory}
                             onChange={(e) => setSubdirectory(e.target.value)}
                           />
+                          {isSubdirectoryValid === false ? (
+                            <p className="text-red-400 mb-2">
+                              Provided sub directory is not valid it should not
+                              start with a / and should end with a /
```

Comment:
Change to:

`The provided subdirectory is not valid. It must end with a <InlineCode>/</InlineCode>, but may not start with one. (e.g. <InlineCode>src/</InlineCode>)`

Reply:
Changed